src/group/api/groups.js

```diff
@@ -66,12 +66,13 @@ export default {
     return axios.delete(`/api/groups/${groupId}/notification_types/${notificationType}/`)
   },
 
-  trustUser (groupId, userId) {
-    return axios.post(`/api/groups/${groupId}/users/${userId}/trust/`)
+  trustUser (groupId, userId, role) {
+    return axios.post(`/api/groups/${groupId}/users/${userId}/trust/`, { role })
   },
 
-  revokeTrust (groupId, userId) {
-    return axios.delete(`/api/groups/${groupId}/users/${userId}/trust/`)
+  revokeTrust (groupId, userId, role) {
+    // passing data in delete(url, config)! luckily, the config object accepts a data property
+    return axios.delete(`/api/groups/${groupId}/users/${userId}/trust/`, { data: { role } })
```

Comment:
I'm a bit mixed about HTTP DELETE with a body, https://stackoverflow.com/questions/299628/is-an-entity-body-allowed-for-an-http-delete-request has mixed opinions, I didn't do a deep dive, but potentially some clients/servers have issues (e.g. someone mentions android not being able to send them), and it might cause a weird issues that's hard to understand.

Just something to keep in mind I think, no change requested.